automated/roles/awx_on_k3s/templates/awx/kustomization.yaml.j2

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: awx
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:
+  - name: awx-secret-tls
+    type: kubernetes.io/tls
+    files:
+      - tls.crt
+      - tls.key
+
+  - name: awx-postgres-configuration
+    type: Opaque
+    literals:
+      - host=awx-postgres-15
+      - port=5432
+      - database=awx
+      - username=awx
+      - password={{ awx_on_k3s_awx_admin_pass }}
+      - type=managed
+
+  - name: awx-admin-password
+    type: Opaque
+    literals:
+      - password={{ awx_on_k3s_awx_admin_pass }}
+
+  # If you want to specify SECRET_KEY for your AWX manually, uncomment following lines and change the value.
+  # Refer AAC documentation for detail about SECRET_KEY.
+  # https://docs.ansible.com/automation-controller/latest/html/administration/secret_handling.html
+  #- name: awx-secret-key
+  #  type: Opaque
+  #  literals:
+  #    - secret_key=MySuperSecureSecretKey123!
```

Comment:
I think this also needs to be a (vaulted) value. (Vault is the task of the user obviously.)

Reply:
"configurable value"